Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_001.HttpAuthorizationRequestHeader.sarif

```diff
@@ -51,7 +51,7 @@
           "message": {
             "id": "Default",
             "arguments": [
-              "SomeAu…",
+              "…111111",
```

Comment:
>111111 [](http://example.com/codeflow?start=16&length=6)

An essential change in behavior, we now prefer the right-most six characters of a randomized token rather than the leftmost. This is because the leftmost portion tends to contain fixed data for many security models (such as databricks keys which are prefixed with dapi_).